exams/management/commands/import_edx_exam_grades.py

```diff
@@ -0,0 +1,81 @@
+"""
+Import proctored exam grades from edx
+"""
+import csv
+import argparse
+from django.contrib.auth.models import User
+
+from django.core.management import BaseCommand, CommandError
+
+from courses.models import Course
+from exams.models import ExamRun, ExamAuthorization
+from exams.pearson.constants import EXAM_GRADE_PASS, EXAM_GRADE_FAIL
+from grades.models import ProctoredExamGrade
+
+
+class Command(BaseCommand):
+    """Parses a csv with exam grades creating or updating ProctoredExamGrade"""
+    help = "Parses a csv with exam grades and creates ProctoredExamGrade"
+
+    def add_arguments(self, parser):
+        parser.add_argument('csvfile', type=argparse.FileType('r'), help='')
+
+        parser.add_argument('course_code', type=argparse.FileType('r'), help='Example: 14.100')
+
+    def handle(self, *args, **kwargs):  # pylint: disable=unused-argument,too-many-locals
+
+        csvfile = kwargs.get('csvfile')
+        course_number = kwargs.get('course_code')
+        reader = csv.DictReader(csvfile.read().splitlines())
```

Comment:
It doesn't look like you need the `read().splitlines()` part here. `DictReader` can read from file handles directly